content/balance/2020-12-08-9.0_FAQ.md

```diff
@@ -352,3 +352,42 @@ No, our dots (Moonfire, Sunfire, Stellar Flare) do NOT snapshot, everything is d
 2 Targets and above:
 - Starfire
 
+<div id="conduits">
+ 
+## [22. What Soulbinds/Conduits should I pick as Night Fae?](#conduits)
+
+</div>
+
+**Soulbinds:**
+- 99% of the time Niya - Poision and the 2nd Potency should be picked over Burrs whenever there are constant 3+ targets
+- 1% of the time Korayn - usable on Council of Blood if you are trying to pad on waiters. Disclaimer: Only usable in Mythic if waiters live long with Niklaus last. Also do not use it while progressing. 
+
+
+
+**Conduits:**
+
+**Potency:**
+- Conflux of the Elements: 100% of the time this should be your first go to potency slot
+- Fury of the Skies: Default go to if the mobs live for a long duration or the pack density is low
+- Umbral Intensity: Default go to if the mobs live for a short duration or the pack density is high
+- Precise Alignment: Around the same value as Fury and Umbral, you can pick it if you dont know which of the other two to pick, might have some value if you want to extend your burst windows on aoe.
+- Stellar Inspiration: Probably not ever worth using with only two Potency slots available
+
+Notes: Conflux is always used and the only choice to be made is between your 2nd potency slot which should be Fury or Umbral most of the time. This is mostly dungeon/personal preference as the have around the same dmg output.
+
+**Endurance:**
+- Well-honed Instincts: Default go to, one of the the most overtuned endurance conduits in the game raid wise.
+- Ursine Vigor: Really good if you will ever go into bear form to soak an ability. Especailly good on progression and if you will ever press Heart of the Wild while being talented into Guardian Affinity. Examples: SLG Goliath wicked blade overlap / Sludgefist 2nd Chain Slam.
+- Tough as Bark: Good if you press Barkskin a lot or the fight timers require a shorter CD on it.
+- Innate Resolve: You can think of this as a buff to Well-Honed Instincts.
+- Layered Mane: Unbelievably useless for Boomie.
+
+Notes: Pick Instincts and default into one of the other three usable conduits as your second choice.
+
+**Finesse:**
+- Born Anew: Good default pick, a static 12 mainstat buff for a friend of yours. Can be used after a wipe is called. It can also be traded in conjunction with Innervate for Power Infusion if a Discipline Priest in present in the raid.
```

Comment:
it's not a static 12 main stat, rather scaling up to (currently) 32 main stat. Also from what I tested the person won't get the buff if the rez is taken after the wipe but needs to be done before combat